scripts/build.js

```diff
@@ -89,14 +89,24 @@ const baseDir = path.join(__dirname, '..');
   );
 
   // copy
-  console.log('copy');
+  console.log('copy bootstrap fonts');
+  await Promise.all(
+    [
+      'node_modules/bootstrap/fonts/glyphicons-halflings-regular.eot',
+      'node_modules/bootstrap/fonts/glyphicons-halflings-regular.svg',
+      'node_modules/bootstrap/fonts/glyphicons-halflings-regular.ttf',
```

Comment:
I'm not sure if these are necessary since `woff` seems to be [supported by all browsers](https://caniuse.com/#feat=woff) (and we only include woff for the other fonts).